### PR TITLE
RNs 36454,36731,36327,35010,36628

### PR DESCRIPTION
**Pull request by EdithLansens**
Hi @Robbe-B and @VeerleLedoux, could you double-check this pull request to verify whether I have correctly added release notes 36454, 36731, 36327, 35010, and 36628 to the user guide? Thanks in advance!

### CODE REVIEW

File: user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_pattern_matching.md [hunk 1]
```diff
@@ -26,3 +28,15 @@ If you are viewing a trend graph for a single parameter, and it contains pattern
...
+
+## Multivariate patterns
+
+From DataMiner 10.3.8/10.4.0 onwards, DataMiner Analytics can automatically recognize recurring patterns in trend graphs showing trend information of multiple parameters. <!-- RN 36454 + 36731 + 36327 -->
+
+When you open a trend graph showing trend information for multiple parameters, you have the option to define a *multivariate trend pattern*.
+
+As with single-parameter patterns (i.e. [univariate patterns](#univariate-patterns)), you can [specify a number of options](xref:Managing_pattern_definitions) in the *Pattern Overview* window. <!-- RN 35010 -->
+
+If you are viewing a trend graph for multiple parameters in which multivariate patterns have been detected, these will be highlighted in orange when you hover the mouse pointer over the button representing a pattern, or if the option *Expand tags* is selected in the right-click menu.
```
Comment (VeerleLedoux) [inline user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_pattern_matching.md:40]: actually you don't necessarily need multiple parameters to be displayed in your trend graph in order to see the occurrences of a multivariate pattern. You need all parameters involved in your multivariate pattern to be loaded in the trend graph to be able to "create" the multivariate pattern. But to just see the detected pattern occurrences at a later time,  one does not necessarily need all the parameters in the graph: one can for instance open the trend graph of only one of the parameters and still see the pattern tags. As explained in your next sentence, by clicking the icon the rest of the parameters can then be loaded if one is interested in seeing which other parameters are involved in the pattern

File: user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_pattern_matching.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ uid: Working_with_pattern_matching
 
 # Working with pattern matching
 
+## Univariate patterns
+
 From DataMiner 10.0.7 onwards, on systems using a Cassandra and Elasticsearch database, DataMiner Analytics can automatically recognize recurring patterns in trend data. For this purpose, you first need to [define the patterns](xref:Defining_a_pattern) DataMiner Analytics needs to track. This pattern definition is also known as a "tag".
```
Comment (VeerleLedoux) [inline user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_pattern_matching.md:9]: wondering if this "Defining a pattern" page also requires a little change as it starts with "In a trend graph showing trend information for one single parameter....": you can now also do it a trend graph showing trend information for multiple parameters, in which case a multivariate pattern will be created

Comment (EdithLansens) [reply user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_pattern_matching.md:9]: Implemented in [commit 956ed0b](https://github.com/SkylineCommunications/dataminer-docs/pull/1699/commits/956ed0bb38e2673a5608f0c9fd379841b7732675).

File: user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_pattern_matching.md [hunk 3]
```diff
@@ -26,3 +28,18 @@ If you are viewing a trend graph for a single parameter, and it contains pattern
...
+From DataMiner 10.3.8/10.4.0 onwards, DataMiner Analytics can automatically recognize recurring patterns in trend graphs showing trend information of multiple parameters. <!-- RN 36454 + 36731 + 36327 -->
+
+When you open a trend graph showing trend information for multiple parameters, you have the option to define a *multivariate trend pattern*.
+
+As with single-parameter patterns (i.e. [univariate patterns](#univariate-patterns)), you can [specify a number of options](xref:Managing_pattern_definitions) in the *Pattern Overview* window. <!-- RN 35010 -->
+
+If you are viewing a trend graph where all parameters involved in the multivariate pattern are loaded and multivariate patterns have been detected, these patterns will be highlighted in orange when you hover the mouse pointer over the button representing a pattern, or if the option *Expand tags* is selected in the right-click menu.
+
+> [!NOTE]
+> You do not need to have multiple parameters displayed in the trend graph to see the detected pattern occurrences.
```
Comment (EdithLansens) [inline user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_pattern_matching.md:43]: Is this more accurate, @VeerleLedoux?

Comment (VeerleLedoux) [reply user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_pattern_matching.md:43]: I suggest to omit the note and changing it into "If you are viewing a trend graph where one or more of the parameters involved in the multivariate pattern are loaded and multivariate patterns have been detected, these patterns will be highlighted in orange when you hover the mouse pointer over the button representing a pattern, or if the option *Expand tags* is selected in the right-click menu."

Comment (EdithLansens) [reply user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_pattern_matching.md]: Implemented in [commit e905b82](https://github.com/SkylineCommunications/dataminer-docs/pull/1699/commits/e905b8213d4b28cecd02a2f186ee38905e44e582). Is this now OK to merge?

**Review by VeerleLedoux — APPROVED** [review]
approved

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**